dash/dash.py

```diff
@@ -1054,5 +1148,9 @@ def run_server(self,
         :return:
         """
         debug = self.enable_dev_tools(debug, dev_tools_serve_dev_bundles)
+        if not debug:
+            # Do not throw debugging exceptions in production.
+            self.config.suppress_validation_exceptions = True
+            self.config.suppress_callback_exceptions = True
```

Comment:
:zap: Only suppress the props validation, we still want to validate the callbacks and it's not an expensive check performance wise.